src/plugins/dam/components/batch-edit-modal/BatchEditFields.jsx

```diff
@@ -0,0 +1,91 @@
+import noop from 'lodash/noop';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Field } from 'redux-form';
+import { Col, FormGroup, Row } from '@triniti/admin-ui-plugin/components';
+import DatePickerField from '@triniti/cms/components/date-picker-field';
+import PicklistPickerField from '@triniti/cms/plugins/sys/components/picklist-picker-field';
+import TextareaField from '@triniti/cms/components/textarea-field';
+import TextField from '@triniti/cms/components/text-field';
+import { DATE_FIELD_QUICK_SELECT_OPTIONS } from '@triniti/cms/plugins/dam/constants';
+
+export default class BatchEditFields extends React.Component {
+  static propTypes = {
+    assetIds: PropTypes.arrayOf(PropTypes.string).isRequired,
+    currentValues: PropTypes.shape({
+      credit: PropTypes.object,
+      expiresAt: PropTypes.instanceOf(Date),
+    }),
+    onBatchUpdate: PropTypes.func,
+    initialValues: PropTypes.shape({}).isRequired,
+  };
+
+  static defaultProps = {
+    currentValues: {},
+    onBatchUpdate: noop,
+  };
+
+  constructor(props) {
+    super(props);
+    this.handleBatchUpdate = this.handleBatchUpdate.bind(this);
+  }
+
+  handleBatchUpdate() {
```

Comment:
@rsumilang when does this get called? far as i can tell the juice is still the button's onClick